Tests/StoreKitUnitTests/TrialOrIntroPriceEligibilityCheckerSK2Tests.swift

```diff
@@ -210,9 +210,11 @@ class TrialOrIntroPriceEligibilityCheckerSK2Tests: StoreKitConfigTestCase {
 
         _ = try await sk2Product.purchase()
 
+        try await Task.sleep(nanoseconds: 100_000_000) // 0.1 second
```

Comment:
I'm not a huge fan of delays, but this test `TrialOrIntroPriceEligibilityCheckerSK2Tests` has been flaky and has failed more than usual when run on iOS 26. We are probably checking intro eligibility too fast right after `purchase()`. This delay seems to reduce the flakiness greatly as it "gives time" for StoreKit testing to notice that the `sk2Product` was indeed purchased.

Reply:
Good point. Added in https://github.com/RevenueCat/purchases-ios/pull/5552/commits/43ed3b2b1ce872570b163d6cc9826d171c84a28e